src/status_im/keycard/backup_key.cljs

```diff
@@ -0,0 +1,36 @@
+(ns status-im.keycard.backup-key
+  (:require [status-im.utils.fx :as fx]
+            [status-im.multiaccounts.recover.core :as multiaccounts.recover]
+            [status-im.navigation :as navigation]
+            [taoensso.timbre :as log]))
+
+(fx/defn backup-card-pressed
+  {:events [:keycard-settings.ui/backup-card-pressed]}
+  [{:keys [db] :as cofx}]
+  (log/debug "[keycard] start backup")
+  (fx/merge cofx
+            {:db (-> db
+                     (assoc-in [:keycard :creating-backup?] true))}
+            (navigation/navigate-to-cofx :seed-phrase nil)))
+
+(fx/defn start-keycard-backup
+  {:events [::start-keycard-backup]}
+  [{:keys [db] :as cofx}]
+  {::multiaccounts.recover/import-multiaccount {:passphrase (get-in db [:multiaccounts/key-storage :seed-phrase])
```

Comment:
The `:passphrase` should be sanitised before sending to import [[example](https://github.com/status-im/status-react/blob/0c55f1208b7eb42f0cb2d77d19caa336f325bcc6/src/status_im/multiaccounts/recover/core.cljs#L165)].  This was a bug in the migration to Keycard PR as well.

Reply:
Oh, thanks for catching this! fixed